lib/widgets/recent_dm_conversations.dart

```diff
@@ -101,8 +133,26 @@ class RecentDmConversationsItem extends StatelessWidget {
               maxLines: 2,
               overflow: TextOverflow.ellipsis,
               title))),
-          const SizedBox(width: 8),
-          // TODO(#253): Unread count
+          unreadCount > 0
+            ? Container(
+                margin: const EdgeInsetsDirectional.only(start: 8, end: 16),
+                child: DecoratedBox(
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(3),
+                    color: const Color.fromRGBO(102, 102, 153, 0.15),
+                  ),
+                  child: Padding(
+                    padding: const EdgeInsetsDirectional.fromSTEB(4, 0, 4, 1),
+                    child: Text(
+                      style: const TextStyle(
+                        fontFamily: 'Source Sans 3',
+                        fontSize: 16,
+                        height: (18 / 16),
+                        fontFeatures: [FontFeature.enable('smcp')], // small caps
+                        color: Color(0xFF222222),
+                      ).merge(weightVariableTextStyle(context)),
+                      unreadCount.toString()))))
+            : const SizedBox(width: 12), // match whitespace at start of row
```

Comment:
Hmm yeah good question. I'll start a quick chat thread to get @terpimost's feedback on this bit — I don't see an answer for it in the Figma design.